README.md

```diff
@@ -12,7 +12,7 @@ Please read the [code of conduct](CODE_OF_CONDUCT.md) and [contribution guidelin
     allprojects {
         repositories {
             google()
-            jcenter()
+            mavenCentral()
```

Comment:
Since we changed this to mavenCentral, shouldn't we also change the sentence above to "Make sure you have **mavenCentral()** repository in your top level **build.gradle** file:"?

Reply:
Yeah, you have right. Thanks. Solved.